### PR TITLE
Release v0.6.1 Patch

### PR DESCRIPTION
**Pull request by ehigham**
This patch inlcudes:
- Fixed docker image generation
- Fixed workload update loop termination

### CODE REVIEW

File: api/Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ WORKDIR /tmp
 
 # Note this step requires docker builds in the root repo context
 # e.g. `docker build -t wfl-api:test -f ./api/Dockerfile .`
-ADD .. .
+ADD . .
```
Comment (rexwangcc) [inline api/Dockerfile:6]: Screen Shot 2021-04-12 at 2 44 04 PM

I'm still puzzled by the issue the Action is reporting, locally, from with the root `make api TARGET=images` works both before and after the change. Let's see if this fixes the Action :octocat:

**Review by rexwangcc — APPROVED** [review]
Thanks for fixing these!

File: api/Dockerfile (hunk 1 above)
Comment (ehigham) [reply api/Dockerfile:6]: Interesting. This failed on my machine. Did the docker build actually run when you tried the before and after?

Comment (rexwangcc) [reply api/Dockerfile:6]: I believe so.. I ran both `make distclean` and `docker system prune -a` before `make api TARGET=images` 🤔 I can double check.

Comment (tbl3rd) [reply api/Dockerfile:6]: How did that ever work then?

**Review by tbl3rd — APPROVED** [review]

File: api/Dockerfile (hunk 1 above)
Comment (ehigham) [reply api/Dockerfile:6]: Honestly I have no idea. Different versions of docker perhaps? I never thought you could add the parent directory.

**Pull request merged**